.github/workflows/trigger-api-template-on-cloud.yml

```diff
@@ -0,0 +1,37 @@
+name: Keep WireMock Cloud Public API Template Library in sync with repo
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+    secrets:
+      ENV_SPECIFIC_SECRET:
+        required: true
+  push:
+    branches:
+      - main
+
+permissions: write-all
+
+jobs:
+  compare_difference_between_branches_and_upgrade_cloud_when_its_different:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: aws-actions/configure-aws-credentials@v2.1.0
+        with:
+          role-to-assume: arn:aws:iam::499333472133:role/GitHub-Manage-WireMock-Api-Templates
+          aws-region: us-east-1
+
+      - uses: aws-actions/amazon-ecr-login@v1
+
+      - run: echo "token=${aws secretsmanager get-secret-value --secret-id wiremock.cloud/live/system | jq -r .SecretString | jq -r .apiToken}" >> $GITHUB_ENV
```

Comment:
Why don't we do this in `sync_changed_templates_with_cloud.sh`? Then we don't need to pass it around in `$GITHUB_ENV` and we can avoid printing it.